README.md

```diff
@@ -1,123 +1,69 @@
-# NXT, standalone part
+# Dawn, a WebGPU implementation
+
+Dawn (formerly NXT) is a open-source and cross-platform implementation of the work-in-progress WebGPU standard.
+It exposes a C/C++ API that maps almost one-to-one to the WebGPU IDL and can be managed as part of a larger system such as a Web browser.
+
+Dawn provides several WebGPU building blocks:
+ - **WebGPU C/C++ headers** that applications and other building blocks use.
+ - **A "native" implementation of WebGPU** using platforms' GPU APIs:
+  - **D3D12** on Windows 10
+  - **Metal** on OSX (and eventually iOS)
+  - **Vulkan** on Windows, Linux (eventually ChromeOS and Android too)
+  - OpenGL as best effort where available
+ - **A client-server implementation of WebGPU** for applications that are in a sandbox without access to native drivers
+
+## Directory structure
+
+- `dawn.json`: the description of the API used to drive code generators.
+- `examples`: some examples showing how Dawn is used.
+- `generator`: code generator for files produces from `dawn.json`
```

Comment:
micro-nit: be consistent in your use of articles in this list: since you used "the" above, use "a" or "the" here.

Reply:
Removed the, double-checked every line.